examples/Getting Started with Semiconductor Module/Python/Getting Started with Semiconductor Module.py

```diff
@@ -0,0 +1,270 @@
+""" Getting Started with Semiconductor Module example!
+
+    Example NI TSM Test Program using simulated niDigital and niDcPower instruments
+"""
+
+import random
+from random import gauss
+
+import nidcpower
+import nidigital
+import nitsm.codemoduleapi
+import pytest
+from nitsm.codemoduleapi import Capability, InstrumentTypeIdConstants
+from nitsm.codemoduleapi import SemiconductorModuleContext
+
+DCPower_options = {"Simulate": True, "DriverSetup": {"Model": "4143", "BoardType": "PXIe"}}
+Digital_options = {"Simulate": True, "driver_setup": {"Model": "6570"}}
+
+
+@pytest.mark.sequence_file("Getting Started with Semiconductor Module.seq")
+def test_getting_started_with_semiconductor_module(system_test_runner):
+    """Test runner."""
+    assert system_test_runner.run()
+
+
+@nitsm.codemoduleapi.code_module
+def open_dcpower_sessions(tsm_context: SemiconductorModuleContext):
+    """Open niDcPower instrument sessions."""
+    resource_strings = tsm_context.get_all_nidcpower_resource_strings()
+    for resource_string in resource_strings:
+        session = nidcpower.Session(resource_string, options=DCPower_options)
+        tsm_context.set_nidcpower_session(resource_string, session)
+
+
+@nitsm.codemoduleapi.code_module
+def open_digital_sessions(tsm_context: SemiconductorModuleContext):
+    """Open niDigital instrument sessions."""
+    instrument_names = tsm_context.get_all_nidigital_instrument_names()
+    for instrument_name in instrument_names:
+        session = nidigital.Session(instrument_name, options=Digital_options)
+        tsm_context.set_nidigital_session(instrument_name, session)
+
+
+@nitsm.codemoduleapi.code_module
+def open_all_instruments_sessions(tsm_context: SemiconductorModuleContext):
+    """Open all instrument sessions."""
+    open_dcpower_sessions(tsm_context)
+    open_digital_sessions(tsm_context)
+
+
+@nitsm.codemoduleapi.code_module
+def continuity(
+    tsm_context: SemiconductorModuleContext,
+    pins,
+):
+    """Measure continuity on configured Pins (niDcPower, niDigital)."""
+    dcpower_filtered_pins = tsm_context.filter_pins_by_instrument_type(
+        pins, InstrumentTypeIdConstants.NI_DCPOWER, Capability.ALL
+    )
+    pin_query, sessions, channel_strings = tsm_context.pins_to_nidcpower_sessions(
+        dcpower_filtered_pins
+    )
+    per_dcpower_session_measurements = []
+    dcpower_continuity_measurements = []
+    for session, channel_string in zip(sessions, channel_strings):
+        assert isinstance(session, nidcpower.Session)
+        assert isinstance(channel_string, str)
```

Comment:
carmentng must have added it for ensuring the data type is right, but it would be redundant if the implementation is correct. I removed the assert section since the data types were verified working during test.